processing/src/main/java/io/druid/segment/NilColumnValueSelector.java

```diff
@@ -85,6 +85,12 @@ public Class classOfObject()
     return Object.class;
   }
 
+  @Override
+  public boolean isNull()
```

Comment:
getLong/Double/Float methods in NilColumnSelector should throw IllegalStateException, if `NullHandling.sqlCompatible()` is true. I suggest to
 1. Remove `final` from this class (it still couldn't be extended, because the constructor is private)
 2. Add private static subclass "SqlCompatible" that throws exceptions
 3. Return the instance of the "main" NilColumnSelector or "SqlCompatible" from instance() method, depending on `NullHandling.sqlCompatible()`

Reply:
done.